irohad/main/impl/peers_file_reader_impl.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "peers_file_reader_impl.hpp"
+
+#include <fstream>
+
+#include "cryptography/public_key.hpp"
+#include "interfaces/common_objects/types.hpp"
+#include "parser/parser.hpp"
+
+using namespace iroha::main;
```

Comment:
Is it safe to use outside the tests source files?

Reply:
Why not? It's a .cpp file and it's not going to be included to anywhere (I hope).